truffle-config.js

```diff
@@ -27,7 +27,10 @@ require("dotenv").config();
 
 module.exports = {
 
-    plugins: ["truffle-security", "solidity-coverage"],
+    plugins: [
+        //"truffle-security",
```

Comment:
Whats the reason for removing?

Reply:
it was not part of the package.json dependencies, and we do not have any workflow that requiring it neither. One can still install truffle-security globally and un-comment the line here for personal inquiry.

Do you think we should keep it anyways?

Reply:
To keep it we need to add it to package.json dependencies.

Reply:
no, not necessary